pkgs/top-level/aliases.nix

```diff
@@ -643,6 +643,7 @@ mapAliases ({
   v8_3_16_14 = throw "v8_3_16_14 was removed in 2019-11-01: no longer referenced by other packages";
   valadoc = throw "valadoc was deprecated on 2019-10-10: valadoc was merged into vala 0.38";
   vamp = { vampSDK = vamp-plugin-sdk; }; # added 2020-03-26
+  vdirsyncerStable  = vdirsyncer; # added 2020-11-08, see https://github.com/NixOS/nixpkgs/issues/103026#issuecomment-723428168
```

Comment:
to have a true deprecation
```suggestion
  vdirsyncerStable  = lib.warn "vdirsyncerStable has been deprecated, please use vdirsyncer" vdirsyncer; # added 2020-11-08, see https://github.com/NixOS/nixpkgs/issues/103026#issuecomment-723428168
```

Reply:
:+1:

Reply:
sorry, ofborg does not like this, and will fail if something is written to stderr

```
$ nix-env -f /home/jon/.cache/nixpkgs-review/pr-103073/nixpkgs -qaP --xml --out-path --show-trace --meta
trace: warning: vdirsyncerStable has been deprecated, please use vdirsyncer
4 packages updated:
khal python37Packages.vdirsyncer (2018-08-05 → 0.16.8) python38Packages.vdirsyncer (2018-08-05 → 0.16.8) vdirsyncer (2018-08-05 → 0.16.8)

2 packages removed:
python3.7-vdirsyncer (†0.16.8) python3.8-vdirsyncer (†0.16.8)
```

Reply:
what you had originally would probably be best

Reply:
Heh.
How's this?

Reply:
as @gebner said, it might be best to just alias it for now.

After the 21.03 release, maybe make it into a throw

Reply:
Hmmm, when you said originally, I thought you meant the first version....
So you mean this, right?
```nix
  vdirsyncerStable  = vdirsyncer; # added 2020-11-08, see https://github.com/NixOS/nixpkgs/issues/103026#issuecomment-723428168
```

Reply:
Yea

Reply:
OK, done.